src/main/java/uk/gov/hmcts/ccd/domain/service/callbacks/CallbackService.java

```diff
@@ -122,14 +128,19 @@ public CallbackService(final SecurityUtils securityUtils,
         Instant startTime = Instant.now();
 
         try {
-            LOG.debug("Invoking callback {}", url);
             final HttpHeaders httpHeaders = new HttpHeaders();
             httpHeaders.add("Content-Type", "application/json");
             if (null != securityHeaders) {
                 securityHeaders.forEach((key, values) -> httpHeaders.put(key, values));
             }
             final HttpEntity requestEntity = new HttpEntity(callbackRequest, httpHeaders);
+            if (logCallbackDetails(url)) {
+                LOG.info("Invoking callback {} of type {} with request: {}", url, callbackType, requestEntity);
+            }
             ResponseEntity<T> responseEntity = restTemplate.exchange(url, HttpMethod.POST, requestEntity, clazz);
+            if (logCallbackDetails(url)) {
+                LOG.info("The callback {} response received: {}", url, responseEntity);
```

Comment:
Same query as above - does logging `responseEntity` itself actually log what we need?

Reply:
yes it does